simplerisk/includes/functions.php

```diff
@@ -56,9 +56,13 @@ function db_open()
     // Connect to the database
     try
     {
-        $GLOBALS['db'] = new PDO("mysql:charset=UTF8;dbname=".DB_DATABASE.";host=".DB_HOSTNAME.";port=".DB_PORT,DB_USERNAME,DB_PASSWORD, array(PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION));
+        $options = array(
```

Comment:
how about setting $options to an empty array (or even put PDO::ATTR_ERRMODE back in there) and then `if(!empty(DB_SSL_CERTIFICATE_PATH))` add it?

Reply:
IIRC from the reading around checking of strings in PHP, the preferred way of doing that would be
```
if(DB_SSL_CERTIFICATE_PATH != '')
```
I'd tried that earlier, but for some reason something wasn't working (but that could have been something else).